.gitignore

```diff
@@ -92,6 +92,7 @@ cpu.out
 /VERSION
 /.air
 /.go-licenses
+/vscode/*
```

Comment:
```suggestion
/.vscode
```

Pretty sure it is with the `.`

Reply:
yup, you're right. thanks for catching :)